yellowbrick/classifier/classification_report.py

```diff
@@ -79,14 +83,17 @@ class ClassificationReport(ClassificationScoreVisualizer):
         Outer dictionary composed of precision, recall, f1, and support scores with
         inner dictionaries specifiying the values for each class listed.
     """
-    def __init__(self, model, ax=None, classes=None, cmap='YlOrRd', **kwargs):
+    def __init__(self, model, ax=None, classes=None, cmap='YlOrRd',
+                 support=None, **kwargs):
         super(ClassificationReport, self).__init__(
             model, ax=ax, classes=classes, **kwargs
         )
 
         self.cmap = color_sequence(cmap)
         self.cmap.set_under(color=CMAP_UNDERCOLOR)
         self.cmap.set_over(color=CMAP_OVERCOLOR)
+        self.support = support
+        self.measurements = ['precision', 'recall', 'f1', 'support']
```

Comment:
Because we have a `SCORE_KEYS` constant, I recommend we change this to:

```python
self._displayed_scores = [key for key in SCORE_KEYS] 
if not support: 
    self._displayed_scores.remove("support") 
```

We can then use this attribute rather than the constant in the `score` method. This will also give us flexibility in the future to change what is displayed and what isn't.

Reply:
I knew you were going to comment on this; I wasn't sure what to do about it.  Thanks !